content/terraform-plugin-testing/v1.13.x/data/plugin-testing-nav-data.json

```diff
@@ -0,0 +1,198 @@
+[
+  {
+    "heading": "Testing"
+  },
+  {
+    "title": "Overview",
+    "path": ""
+  },
+  {
+    "title": "Acceptance Testing",
+    "routes": [
+      {
+        "title": "Overview",
+        "path": "acceptance-tests"
+      },
+      {
+        "title": "Test Cases",
+        "path": "acceptance-tests/testcase"
+      },
+      {
+        "title": "Test Steps",
+	"routes": [{
+		"title": "Overview",
+		"path": "acceptance-tests/teststep"
+	}, {
+		"title": "Lifecycle mode",
+		"path": "acceptance-tests/lifecycle-mode"
+	}, {
+		"title": "Import mode",
+		"path": "acceptance-tests/import-mode"
+	}, {
+		"title": "Refresh mode",
```

Comment:
TODO: remove Refresh mode from the nav. We do not have a standalone page for this yet.